intl/msg_hash_us.h

```diff
@@ -1437,9 +1437,17 @@ MSG_HASH(
    MENU_ENUM_SUBLABEL_CRT_SWITCH_PORCH_ADJUST,
    "Cycle through these options to adjust the porch settings to change the image size."
    )
+MSG_HASH(
+   MENU_ENUM_LABEL_VALUE_CRT_SWITCH_HIRES_MENU,
+   "Use high resolution menu"
+   )
+MSG_HASH(
+   MENU_ENUM_SUBLABEL_CRT_SWITCH_HIRES_MENU,
+   "Use high resolution menu"
+   )
```

Comment:
Please don't create redundant sublabels. Please try to describe the function and/or purpose and/or when/how to use this function in more detail. Either make the sublabel be useful, in any way, or don't add one. Thank you.